x-pack/plugin/watcher/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/painless/40_exception.yml

```diff
@@ -38,8 +38,10 @@
 ---
 "Test painless exceptions are returned when logging a broken response":
   - skip:
-      version: "8.7.0 - 8.7.1"
-      reason:  "self-referencing objects were in Painless instead of Mustache in 8.7.0 to 8.7.1"
+      known_issues:
+        - cluster_feature: "gte_v8.7.0"
+          fixed_by: "gte_v8.7.2"
+      reason: "self-referencing objects were in Painless instead of Mustache in 8.7.0 to 8.7.1"
```

Comment:
Nit: reading the reason, I think this may be better represented by a pair of tests, as it does not look like a bug. But not fussy about this.

Reply:
Hmm, to me this reads like a bug.
Anyways, goal of the PR is to translate version ranges to known_issues / awaits_fix. I'm not willing to expand the scope here ;)